### PR TITLE
Rename CacheClass to ClassByNameCache to avoid OpenJDK conflict

### PR DESCRIPTION
**Pull request by pshipton**
See https://github.com/ibmruntimes/openj9-openjdk-jdk17/pull/99
https://github.com/ibmruntimes/openj9-openjdk-jdk/pull/388

Signed-off-by: Peter Shipton <Peter_Shipton@ca.ibm.com>

### CODE REVIEW

**Comment by pshipton** [comment]
jenkins compile alinux64 jdk8

**Comment by pshipton** [comment]
jenkins compile xmac jdk8

File: closed/adds/jdk/src/share/classes/java/io/ClassByNameCache.java [hunk 1]
```diff
@@ -30,19 +30,19 @@
...
-/* ClassCache is Primarily responsible for Caching the results of the className lookups and hence to avoid 
+/* ClassByNameCache is Primarily responsible for Caching the results of the className lookups and hence to avoid
  * multiple Lookup for same Class Instance.
- * ClassCache provides a ConcurrentHash based ClassCache which is looked up prior to calling the class.forName
+ * ClassByNameCache provides a ConcurrentHash based ClassByNameCache which is looked up prior to calling the class.forName
  *  Method resolveClass() from ObjectInputStream uses this Cache.
- *  
+ *
  *  Caching is done only when the actually used loader for a Class is one of the Sytem loaders (ie) App Class Loader,
  *  System Extension loader and BootStrap loader
```
Comment (keithc-ca) [inline closed/adds/jdk/src/share/classes/java/io/ClassByNameCache.java:39]: Please finish this sentence with a period.

File: closed/adds/jdk/src/share/classes/java/io/ClassByNameCache.java [hunk 2]
```diff
@@ -146,7 +146,7 @@ private Object createEntry(CacheKey key) {
         return value;
     }
 /*
- * This is the entry point in to the cache from ObjectInputStream. First Lookup is done based on the className and Loader 
+ * This is the entry point in to the cache from ObjectInputStream. First Lookup is done based on the className and Loader
```
Comment (keithc-ca) [inline closed/adds/jdk/src/share/classes/java/io/ClassByNameCache.java:149]: Please fix the indentation on line 156.

**Review by keithc-ca — CHANGES_REQUESTED** [review]

**Comment by pshipton** [comment]
Fixed the 2 review comments.

**Comment by keithc-ca** [comment]
There's no need to repeat a build for those cosmetic changes.

**Pull request merged**